src/samay.js

```diff
@@ -110,6 +110,20 @@ function parseDateTime5(strDate) {
 		}
 }
 
+function parseDateTime6(strDate) {
```

Comment:
But then the name should be something about the function. 
```parseDateTime6``` does not make sense